extensions/Worker.Extensions.Http/src/FromBodyConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.Converters
+{
+    internal class FromBodyConverter : IInputConverter
```

Comment:
Are we missing the piece of code to register this new input converter?

Reply:
I think that because `[FromBody]` is an `InputConverterAttribute`, it gets auto-used.

Reply:
Correct. This is automatically used as it is an `InputConverterAttribute`

Reply:
Oh yes, I forgot that.